pkg/products/grafana/reconciler.go

```diff
@@ -90,553 +51,51 @@ func NewReconciler(configManager config.ConfigReadWriter, installation *integrea
 		installation:  installation,
 		mpm:           mpm,
 		log:           logger,
-		Reconciler:    resources.NewReconciler(mpm).WithProductDeclaration(*productDeclaration),
+		Reconciler:    resources.NewReconciler(mpm),
 		recorder:      recorder,
 	}, nil
 }
 
 func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1alpha1.RHMI, productStatus *integreatlyv1alpha1.RHMIProductStatus, client k8sclient.Client, productConfig quota.ProductConfig, uninstall bool) (integreatlyv1alpha1.StatusPhase, error) {
 	r.log.Info("Start Grafana reconcile")
 
-	operatorNamespace := r.Config.GetOperatorNamespace()
-	productNamespace := r.Config.GetNamespace()
-
-	phase, err := r.ReconcileFinalizer(ctx, client, installation, string(r.Config.GetProductName()), uninstall, func() (integreatlyv1alpha1.StatusPhase, error) {
-		phase, err := resources.RemoveNamespace(ctx, installation, client, productNamespace, r.log)
-		if err != nil || phase != integreatlyv1alpha1.PhaseCompleted {
-			return phase, err
-		}
+	requestsPerUnitStr := fmt.Sprint(productConfig.GetRateLimitConfig().RequestsPerUnit)
+	activeQuota := productConfig.GetActiveQuota()
 
-		phase, err = resources.RemoveNamespace(ctx, installation, client, operatorNamespace, r.log)
+	if !resources.IsInProw(installation) {
+		// Creates Grafana RateLimit ConfigMap
+		phase, err := ReconcileGrafanaRateLimmitDashboardConfigMap(ctx, client, r.installation, requestsPerUnitStr, activeQuota)
 		if err != nil || phase != integreatlyv1alpha1.PhaseCompleted {
+			events.HandleError(r.recorder, installation, phase, "Failed to reconcile Grafana Ratelimit Dashboard ConfigMap", err)
 			return phase, err
 		}
-
-		if err = r.deleteConsoleLink(ctx, client); err != nil {
```

Comment:
we need to have console link + we need to remove the console link. This can be either done by PO or here

Reply:
done. Thank you.
Link was verified in Cluster web-console
![Screenshot from 2023-10-15 14-16-45](https://github.com/integr8ly/integreatly-operator/assets/56625811/efc55c6b-d9e7-45ab-986a-f66a85740bce)